security/tls.go

```diff
@@ -116,18 +116,18 @@ func LoadInsecureTLSConfig() *tls.Config {
 	return nil
 }
 
-// LoadClientTLSConfigFromDir creates a client TLSConfig by loading the root CA certs from the
-// specified directory. The directory must contain the following files:
+// LoadClientTLSConfig creates a client TLSConfig by loading the CA and client certs
+// from the specified directory. The directory must contain the following files:
 // - ca.crt   -- the certificate of the cluster CA
-// - node.client.crt -- the client certificate of this node; should be signed by the CA
-// - node.client.key -- the certificate key
+// - <username>.client.crt -- the client certificate of this client; should be signed by the CA
+// - <username>.client.key -- the certificate key
 // If the path is prefixed with "embedded=", load the embedded certs.
-func LoadClientTLSConfigFromDir(certDir string) (*tls.Config, error) {
-	certPEM, err := readFileFn(path.Join(certDir, "node.client.crt"))
+func LoadClientTLSConfig(certDir, username string) (*tls.Config, error) {
+	certPEM, err := readFileFn(path.Join(certDir, clientCertFile(username)))
```

Comment:
I guess that's only relevant if this is somehow in the code path of incoming client requests, which it probably isn't.